code-submissions-clustering-core/src/test/kotlin/org/jetbrains/research/code/submissions/clustering/cluster/ClusteringTest.kt

```diff
@@ -218,19 +218,26 @@ class ClusteringTest : ParametrizedBaseWithUnifierTest(getTmpProjectDir()) {
 
         @JvmStatic
         fun getTestDataFromBin(): List<Arguments> {
-            val binDir = "${Paths.get("").toAbsolutePath()}/src/test/resources/bin.data"
-            val serializationDirs =
-                File(binDir).list { dir, name -> File(dir, name).isDirectory }.map { Paths.get("$binDir/$it") }
-            val submissionsGraphs = serializationDirs.map { it.toSubmissionsGraph() }
-            val arguments = mutableListOf<Arguments>()
+            val binDirName = "bin.data"
+            val classLoader = Thread.currentThread().contextClassLoader
+            val binDir = classLoader.getResource(binDirName)?.toURI()?.let { Paths.get(it) }
```

Comment:
Probably you should return an empty list here if the binDir is null and next work with a non nullable va;ue

Reply:
I will fix it